setup-charm.sh

```diff
@@ -59,3 +59,6 @@ if [ -f "requirements.txt" ]; then
   pip install -r requirements.txt
 fi
 
+# configure dynamic linker run-time bindings
+sudo ldconfig
```

Comment:
Looks good. Will we have to do same changes in setup-charm-homebrew.sh file as well?